lib/Conversion/FIRRTLToRTL/LowerToRTL.cpp

```diff
@@ -1444,7 +1444,9 @@ LogicalResult FIRRTLLowering::visitDecl(RegResetOp op) {
 
   Value clockVal = getLoweredValue(op.clockVal());
   Value resetSignal = getLoweredValue(op.resetSignal());
-  Value resetValue = getLoweredValue(op.resetValue());
+  // Reset values may be narrower than the register.  Extend appropriately.
+  Value resetValue = getLoweredAndExtOrTruncValue(
```

Comment:
Can this use `getLoweredAndExtendedValue`?  I don't think we want to support implicit truncations here.